modules/dnn/src/darknet/darknet_io.cpp

```diff
@@ -556,20 +556,21 @@ namespace cv {
                     {
                         int kernel_size = getParam<int>(layer_params, "size", -1);
                         int pad = getParam<int>(layer_params, "pad", 0);
+                        int padding = getParam<int>(layer_params, "padding", 0);
```

Comment:
What if `padding > 0` ?

Reply:
if pad=1 then padding will be assigned (kernel_size/2). if pad=0 or pad is not mentioned then padding will be manual i.e. whatever mentioned in model and if padding and pad both are not mentioned then padding will be assigned 0.